trapper/common/utils.py

```diff
@@ -82,3 +82,20 @@ def add_property(inst, name_to_method: Dict[str, Callable]):
 def is_equal(x: Union[Dict, List], y: Union[Dict, List]) -> bool:
     """Checks equality of two nested container type e.g. list or dict"""
     return not DeepDiff(x, y)
+
+
+def merge_args_safe(
```

Comment:
I don't think if should belong in this module if it is only used in `trapper/commands.py`